mlflow/server/js/src/common/sdk/PipelinesService.js

```diff
@@ -0,0 +1,11 @@
+import { getJson } from '../utils/FetchUtils';
+
+export class PipelinesService {
```

Comment:
This file is used in feature-store UI.

Reply:
This class is used in feature-store UI. I'll update the copybara config to ignore this file.